sdk/ml/azure-ai-ml/azure/ai/ml/operations/_workspace_hub_operation.py

```diff
@@ -113,14 +109,15 @@ def begin_create(
         Returns the WorkspaceHub if already exists.
 
         :keyword workspace_hub: WorkspaceHub definition.
-        :paramtype workspace_hub: WorkspaceHub
+        :paramtype workspace_hub: ~azure.ai.ml.entities.WorkspaceHub
         :keyword update_dependent_resources: Whether to update dependent resources. Defaults to False.
         :paramtype update_dependent_resources: boolean
         :return: An instance of LROPoller that returns a WorkspaceHub.
         :rtype: ~azure.core.polling.LROPoller[~azure.ai.ml.entities.WorkspaceHub]
         """
 
         def get_callback():
+            """callback to called after completion"""
```

Comment:
Could we reword this? I'm assuming this is a typo and should be "Callback to be called after completion"